src/mono/mono/mini/mini-llvm.c

```diff
@@ -13546,6 +13563,10 @@ add_intrinsic (EmitContext *ctx, int id)
 						 */
 						 LLVMTypeRef associated_type = intrin_types [vw][0];
 						 intrins = add_intrins2 (module, id, distinguishing_type, associated_type, &intrins_type);
+					} else if (kind == INTRIN_kind_add_pointer) {
```

Comment:
what is this change about and why we have to track this separately? Mono supported APIs that takes pointers, right?

Reply:
This is related to calling LLVM intrinsics. Yes, Mono supports pointer type.